src/pygeosx/PyGroup.cpp

```diff
@@ -46,6 +47,16 @@ struct PyGroup
   dataRepository::Group * group;
 };
 
+struct PySolver
+{
+  PyObject_HEAD
+
+  static constexpr char const * docString =
+    "A Python interface to geosx::SolverBase.";
+
+  geosx::SolverBase * group;
+};
```

Comment:
This seems a little backwards to me. I feel like `PyGroup.cpp` shouldn't have to include anything from `pysolver`. It also feels strange to define `PySolver` in this file.

Reply:
Yes that's actually something I noticed yesterday and removed. It was from a previous version.